src/main/java/ch/njol/skript/expressions/ExprSpawn.java

```diff
@@ -38,75 +31,83 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.bukkit.event.world.SpawnChangeEvent;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Spawn")
 @Description("The spawn point of a world.")
-@Examples({"teleport all players to spawn",
-		"set the spawn point of \"world\" to the player's location"})
+@Examples({
+	"teleport all players to spawn",
+	"set the spawn point of \"world\" to the player's location"
+})
 @Since("1.4.2")
 public class ExprSpawn extends PropertyExpression<World, Location> {
+
 	static {
-		Skript.registerExpression(ExprSpawn.class, Location.class, ExpressionType.PROPERTY, "[the] spawn[s] [(point|location)[s]] [of %worlds%]", "%worlds%'[s] spawn[s] [(point|location)[s]]");
+		Skript.registerExpression(ExprSpawn.class, Location.class, ExpressionType.PROPERTY,
+			"[the] spawn[s] [(point|location)[s]] [of %worlds%]",
+			"%worlds%'[s] spawn[s] [(point|location)[s]]"
+		);
 	}
-	
-	@SuppressWarnings({"unchecked", "null"})
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```

Comment:
The init method isn't needed

Reply:
I originally thought of that but this class is PropertyExpression not SimplePropertyExpression so that method doesn't already setExpr in super